README.md

```diff
@@ -25,7 +25,7 @@ The BUGTRONS 2.0, revamped version of [BUGTRONS](https://github.com/devstrons/bu
  
 - [DEVSTRONS'](https://devstrons.bio.link/): Student-run Open source/ project-based, inclusive & diverse community intending to Learn, Code and Tron (grow).
 
-- [Learn With Community](): Let's Learn, grow and give back to this Community.
+- [Learn With Community](http://learnwithcommunity.bio.link): A tech community, focusing on the best way of learn, learning together.
```

Comment:
```suggestion
- [Learn With Community](https://learnwithcommunity.bio.link): A tech community, focusing on the best way to learn, learning together.
```

Reply:
@krishguptadev, have you discussed with LWC Core to change this description in Bugtrons, as the present description is Tagline of LWC?

Reply:
Yes, these changes are been discussed with Aditya.